### DISCUSSION TITLE
Prisma schema not loading from .env.local in Vercel Full Stack tutorial

### DISCUSSION THREAD

**Discussion by kychow** [vercel/community#2557] [Help] [answered]
In Step 3 of [How to Build a Fullstack App with Next.js, Prisma, and Vercel Postgres](https://vercel.com/guides/nextjs-prisma-postgres#step-2:-set-up-your-vercel-postgres-database), initiating a Postgres DB instance with `npx prisma db push` fails because the prisma DB schema can't read the `POSTGRES_URL_NON_POOLING` variable from `.env.local` file.

```
> npx prisma db push
Prisma schema loaded from schema.prisma
Datasource "db": PostgreSQL database

Error: Prisma schema validation - (get-config wasm)
Error code: P1012
error: Environment variable not found: POSTGRES_URL_NON_POOLING.
  -->  schema.prisma:11
   | 
10 |   url = env("POSTGRES_PRISMA_URL") // uses connection pooling
11 |   directUrl = env("POSTGRES_URL_NON_POOLING") // uses a direct connection
   | 

Validation Error Count: 1
[Context: getConfig]

Prisma CLI Version : 4.14.1
````

However, it seems to read the `POSTGRES_PRISMA_URL` variable in the line above it just fine. 

I have visually verified the variable exists in `.env.local` and the prisma is directly copied from the tutorial docs: 
```
// schema.prisma

generator client {
  provider = "prisma-client-js"
  previewFeatures = ["jsonProtocol"]
}

datasource db {
  provider = "postgresql"
  url = env("POSTGRES_PRISMA_URL") // uses connection pooling
  directUrl = env("POSTGRES_URL_NON_POOLING") // uses a direct connection
  shadowDatabaseUrl = env("POSTGRES_URL_NON_POOLING") // used for migrations
}

model Post {
  id        String     @default(cuid()) @id
  title     String
  content   String?
  published Boolean @default(false)
  author    User?   @relation(fields: [authorId], references: [id])
  authorId  String?
}

model User {
  id            String       @default(cuid()) @id
  name          String?
  email         String?   @unique
  createdAt     DateTime  @default(now()) @map(name: "created_at")
  updatedAt     DateTime  @updatedAt @map(name: "updated_at")
  posts         Post[]
  @@map(name: "users")
}
```
My github repo: https://github.com/kychow/next_blog 

Has anyone run into a similar issue before? I can't imagine why `POSTGRES_PRISMA_URL` is being read from `.env.local` but `POSTGRES_URL_NON_POOLING` isn't.

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing this bug, @kychow. If you rename the `.env.local` to simply `.env` it should unblock you. I'll let the team know we need to update the guide 🙇

**Reply by kychow** [reply]
Thanks amy! Issue has been resolved

**Reply by nimeshmaharjan1** [reply]
but how do i do it in vercel?

**Comment by janpio** [comment]
Also, note that the `shadowDatabaseUrl` is not needed any more in your `datasource` as well: https://github.com/orgs/vercel/discussions/2515#discussioncomment-6450133 Best just delete that line, things will still work.
